base/constants.go

```diff
@@ -230,6 +230,13 @@ func UnitTestUrlIsWalrus() bool {
 	return ServerIsWalrus(UnitTestUrl())
 }
 
+func TestsRequireBootstrapConnection() bool {
+	if !TestUseCouchbaseServer() && UnitTestUrlIsWalrus() {
+		return true
+	}
+	return false
+}
```

Comment:
nit, shortens code below, note rename to match other test functions - let's move this to `util_testing.go`.

```suggestion
func TestRequiresBootstrapConnection(t *testing.T) {
	if UnitTestUrlIsWalrus() {
		t.Skipf("Tests requiring a bootstrap connection are not supporting using rosmar yet - CBG-3271")
	}
}
```